e2e_testing/xfail_sets.py

```diff
@@ -904,6 +940,9 @@
     "ViewCollapseDynamicWithAtenSizeIntModule_basic",
     "UnsafeViewCollapseDynamicWithAtenSizeIntModule_basic",
 
+    # LLVM ERROR: SmallVector unable to grow. Requested capacity (9223372036854775808) is larger than maximum value for size type (4294967295)
```

Comment:
Do you know what is causing this regression? We should try to avoid having new changes introduce crashes.

Reply:
No crash anymore, I will just xfail it.